src/v2/Apps/_About2/AboutApp.tsx

```diff
@@ -18,235 +27,406 @@ export const AboutApp: React.FC = () => {
-                  interests, and current collection in mind. Your specialist can
-                  also source works which aren't publicly listed on the site
-                  through our international network of gallery partners.
+const CollectorInfoSection: React.FC = () => {
+  const image = resized(
+    "http://files.artsy.net/about2_page_collector_img.jpg",
+    {
+      width: 910,
+      height: 652,
+    }
+  )
+
+  return (
+    <GridColumns>
+      <Column span={12}>
+        <Text variant="xl" my={6}>
+          Collecting
+        </Text>
+      </Column>
+      <Column span={6}>
+        <ResponsiveBox aspectHeight={652} aspectWidth={910} maxWidth="100%">
+          <Image
+            src={image.src}
+            width="100%"
+            height="100%"
+            srcSet={image.srcSet}
+            lazyLoad
+            alt=""
+          />
+        </ResponsiveBox>
+        <Text variant="xs" textColor="black60">
+          The Collection fo Carole Server by Emily Johnston for Artsy 2015.
+          Courtesy of Carole Server.
+        </Text>
+      </Column>
+
+      <Column span={6}>
+        <Text variant="xs" textTransform="uppercase">
+          On Artsy
+        </Text>
+        <Text variant="xl" mt={1}>
+          Where Collectors Belong
+        </Text>
+        <Text variant="sm" mt={1}>
+          Artsy lets collectors find their favorite artists—and discover some
+          new ones—all in one place. We bring together over 1 million artworks
+          from our network of 4,000+ partners, including the world’s leading
+          galleries, auction houses, fairs, and institutions. To connect
+          collectors to the art they care about, we feature the largest
+          selection of international emerging, established, and blue-chip
+          artists.
+        </Text>
+      </Column>
+      <Column span={12}>
+        <Spacer my={6} />
+        <Text borderBottom="1px solid" borderBottomColor="black15" />
+        <Spacer my={6} />
+      </Column>
+    </GridColumns>
+  )
+}
+
+const ArtsySpecialistsSection: React.FC = () => {
+  return (
+    <GridColumns>
+      <Column span={6}>
+        <Text variant="xs" textTransform="uppercase">
+          Artsy Specialists
+        </Text>
+        <Text variant="xl" mt={1}>
+          Here to Help You Find the Art You Love
+        </Text>
+        <Text variant="sm" mt={1}>
+          Artsy specialists are available to all Artsy members, wherever you are
+          in your collector’s journey. Our specialists join Artsy from leading
+          auction houses, galleries, and museums, and are ready to help you find
+          works that you love—with your price range, taste, and current
+          collection in mind.
+        </Text>
+        <Text variant="sm" mt={4}>
+          More questions about collecting?
+        </Text>
+        <Flex>
+          <Button
+            variant="primaryBlack"
+            width="40%"
```

Comment:
I find any kind of explicit width or height suspicious — we could likely achieve the desired layout using `GridColumns` so that we adhere to the underlying grid system.

Reply:
I haven't looked at the comps but it _looks_ to me like we'd want the buttons to just occupy the full space with a `space.2` gap in between.

So just removing the widths and using `flex={1}` on each button and sticking a `<Spacer ml={2} />` in between should accomplish this.